include/zephyr/bluetooth/mesh/access.h

```diff
@@ -187,6 +187,10 @@ struct bt_mesh_elem {
 #define BT_MESH_MODEL_ID_REMOTE_PROV_SRV           0x0004
 /** Remote Provisioning Client */
 #define BT_MESH_MODEL_ID_REMOTE_PROV_CLI           0x0005
+/** Bridge Configuration Sever */
+#define BT_MESH_MODEL_ID_BRG_CFG_SRV               0x0008
+/** Bridge Configuration Client */
```

Comment:
```suggestion
/** Subnetwork Bridge Configuration Client */
```

Reply:
Will keep the name from the Mesh Protocol spec.